.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,7 @@
+**What problem is this solving?**
+
+<!--- What is the motivation and context for this change? -->
+
+**How should this be manually tested?**
+
+**Screenshots or example usage:**
```

Comment:
we actually use a different PR template for the repos that @vtex-apps/checkout-ui owns, can you adjust them to that? [this is the template](https://raw.githubusercontent.com/vtex-apps/checkout/master/.github/PULL_REQUEST_TEMPLATE.md).

Reply:
Done